README.md

```diff
@@ -23,23 +23,9 @@ In order to build `libremarkable` and the examples (`spy.so` and `demo`), you'll
 
 You can then set up your Rust toolchain for cross compilation with: `rustup target add armv7-unknown-linux-gnueabihf`.
 
-Once that's done, you should add the following to `.cargo/config` (replace `<path-to-installed-oecore-toochain>` with the directory you installed the Remarkable toolchain to):
-```
-[target.armv7-unknown-linux-gnueabihf]
-linker = "<path-to-the-installed-oecore-toolchain>/sysroots/x86_64-oesdk-linux/usr/bin/arm-oe-linux-gnueabi/arm-oe-linux-gnueabi-gcc"
-rustflags = [
-  "-C", "link-arg=-march=armv7-a",
-  "-C", "link-arg=-marm",
-  "-C", "link-arg=-mfpu=neon",
-  "-C", "link-arg=-mfloat-abi=hard",
-  "-C", "link-arg=-mcpu=cortex-a9",
-  "-C", "link-arg=--sysroot=<path-to-the-installed-oecore-toolchain>/sysroots/cortexa9hf-neon-oe-linux-gnueabi",
-]
-```
-
-(`<path-to-the-installed-oecore-toolchain` will likely be `/usr/local/oecore-x86_64/`, if you did the default install on Linux.)
-
-If you have further questions, feel free to ask in Issues.
+In order for rust to leverage the toolchain a `.cargo/config` file is required. This file can be generated using the `gen_cargo_config.py`. First the toolchain environment must be
+sourced. It location is can be found within the toolchain installation directory. The correct path is also referenced in the toolchain [wiki](https://remarkablewiki.com/devel/toolchain).
+After the environment is loaded the script will read the environment variables to generate the correct `.cargo/config` file for your toolchain.
```

Comment:
I suggest adding an example here to, like:

> For example, when run against toolchain version 1.2.3, this might generate:
>
> ```
> <config file text>
> ```

Aside from documenting the format, this may be useful to folks who don't want to or can't run the script.